xdebug_trace_computerized.c

```diff
@@ -226,6 +226,50 @@ void xdebug_trace_computerized_function_return_value(void *ctxt, function_stack_
 	xdfree(str.d);
 }
 
+void xdebug_trace_computerized_assignment(void *ctxt, function_stack_entry *fse, char *full_varname, zval *retval, char *op, char *filename, int lineno TSRMLS_DC)
+{
+	xdebug_trace_computerized_context *context = (xdebug_trace_computerized_context*) ctxt;
+	xdebug_str str = XDEBUG_STR_INITIALIZER;
+	unsigned int j = 0;
+	char *tmp_str;
+
+	xdebug_str_add(&str, xdebug_sprintf("%d\t", fse->level), 1);
+    /* no function_nr */
+	xdebug_str_add(&str, "\t", 0);
+
+	tmp_str = xdebug_show_fname(fse->function, 0, 0 TSRMLS_CC);
+
+	xdebug_str_add(&str, "2\t", 0);
```

Comment:
Instead of "2", I suggest to use either "=" or "A". Return values already use "R" so it would be more in line with that.